### DISCUSSION TITLE
epoche increased fore php8.0 / 7.4

### DISCUSSION THREAD

**Discussion by jonashrem** [oerdnj/deb.sury.org#1801] [Q&A] [answered]
Hi,

I'm wondering why see epoche of the php8.0 and 7.4 packages has been increased to 1.

```
Package: php8.0
Version: 1:8.0.20-1+0~20220614.37+debian10~1.gbp11fdd0
```
This seems to be a change since 8.0.19, but I don't see any reason for this

**Comment by jonashrem** [comment, marked as answer]
Oh, sorry. I see it's already answered here: https://github.com/oerdnj/deb.sury.org/discussions/1798

**Reply by oerdnj** [reply]
Yeah, unfortunately. I am not happy about it either, in fact, the opposite of happy could be used to describe my emotions towards this change. I've discovered it post fact I'm otherwise I would have tried to find a better solution that would work for Canonical  and not break the rest of the world at the same time.

**Reply by jonashrem** [reply]
Yes, that's understandable.
